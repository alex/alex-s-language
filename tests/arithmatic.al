print(3+4)
print(2)
a = b = 3
print(a)
print(a+b)
print(a-b)
print(a*b)
print(a/b)
print(a**b)
print(-3**2)
print(
    3)
