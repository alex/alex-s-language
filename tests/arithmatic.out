7
2
3
6
0
9
1
27
-9
